cmd/flags/flags.go

```diff
@@ -36,8 +37,25 @@ const (
 	logLevel                       = "log-level"
 	dependencyStorageType          = "dependency-storage.type"
 	dependencyStorageDataFrequency = "dependency-storage.data-frequency"
+	configFile                     = "config-file"
 )
 
+// AddConfFileFlag adds flags for ExternalConfFlags
+func AddConfFileFlag(flagSet *flag.FlagSet) {
+	flagSet.String(configFile, "", "Configuration file in JSON, TOML, YAML, HCL, or Java properties formats (default none). See spf13/viper for precedence.")
+}
+
+// TryLoadConfigFile initializes viper with config file specified as flag
+func TryLoadConfigFile(v *viper.Viper, logger *zap.Logger) {
+	if file := v.GetString(configFile); file != "" {
+		v.SetConfigFile(file)
+		err := v.ReadInConfig()
+		if err != nil {
+			logger.Fatal(fmt.Sprintf("Fatal error config file: %s \n", file), zap.Error(err))
```

Comment:
Nit: don't use string formating. File name can be passed as a string field. Message could be "error loading config".